snowfakery/data_generator_runtime_object_model.py

```diff
@@ -19,7 +19,7 @@
 # objects that represent the hierarchy of a data generator.
 # roughly similar to the YAML structure but with domain-specific objects
 Definition = Union["ObjectTemplate", "SimpleValue", "StructuredValue"]
-FieldValue = Union[None, Scalar, ObjectRow, tuple, PluginResult, ObjectReference]
+FieldValue = Union[None, Scalar, ObjectRow, tuple, PluginResult, ObjectReference, list, dict]
```

Comment:
Since there are other examples that will fail, how about adding support for more native types?:
```suggestion
FieldValue = Union[
    None, bool, int, float, str, bytes, list, dict, tuple, set,
    Scalar, ObjectRow, PluginResult, ObjectReference
]
```